src/coreclr/inc/regdisp.h

```diff
@@ -155,7 +155,7 @@ inline TADDR GetRegdisplayStackMark(REGDISPLAY *display) {
 
 #elif defined(TARGET_64BIT)
 
-#if defined(TARGET_ARM64)
+#if defined(TARGET_ARM64) || defined(TARGET_ARM)
```

Comment:
This does not look right. `TARGET_64BIT` above should not be defined for `TARGET_ARM`. 32-bit ARM has very different volatile context pointers.